packages/shared/src/extension/__tests__/app.spec.ts

```diff
@@ -0,0 +1,194 @@
+import axiosMock from '../../test/__mocks__/axios.mock';
+import fetchMock from 'jest-fetch-mock';
+
+process.env.API_ROOT = 'http://localhost:9001/api';
```

Comment:
note, it this is shared, should not only refer to youtube (port 9000)

Reply:
the value assigned here is irrelevant as long is a string, cause in the `*.spec.ts` tests everything is now mocked. So API calls are not performed for real